test/caddy/docker-compose.yaml

```diff
@@ -3,8 +3,8 @@ services:
     image: xxxtest/caddy
     build: .
     ports:
-      - 8080:80
-      - 8443:443
+      - "8080:80"
```

Comment:
not needed in THIS case but it's still good practice

Reply:
Yeah, good catch, I've internalized YAML quoting rules but I accept that most people haven't had the need to lol